spec/reek/smells/unused_private_method_spec.rb

```diff
@@ -4,6 +4,12 @@
 require_relative 'smell_detector_shared'
 
 RSpec.describe Reek::Smells::UnusedPrivateMethod do
+  let(:configuration) do
```

Comment:
Sadly enough, due to the way we load SmellDetectors and Examiners something like:

```Ruby
  before do
    allow(described_class).
      to receive(:default_config).
      and_return(described_class.superclass.default_config)
  end
````

or

```Ruby
  before do
    described_class.instance_eval { def default_config; super; end }
  end
```

but given that we don't do this every other week I'd consider this good enough given the task at hand.